src/N3Lexer.js

```diff
@@ -471,7 +485,7 @@ export default class N3Lexer {
           }
           // Otherwise, tokenize as far as possible
           else {
-            this._input += data;
+            this._input += this._readStartingBom(String(data));
```

Comment:
Is this `String` constructor necessary?

Reply:
Yes otherwise it will fail with `Uncaught TypeError: input.startsWith is not a function`.

Reply:
What type if `input` then if that happens?  It's supposed to be a string already. (Buffer?)

Reply:
Apparently there are 7 tests where typeof input === Object

Reply:
I think `Buffer`. Will have a look myself. Could you just add a test case for `streamOf(['', ''\ufeff''])`? Then I'll finish up. Thanks!

Reply:
Done. I fear I pushed changes to the package-lock.json file. Look out for that!